src/plot_erpimage.jl

```diff
@@ -128,9 +128,7 @@ function plot_erpimage!(
             xautolimitmargin = (0, 0),
             sub_config1.axis...,
```

Comment:
it does make any sense to add here the axis, the fields already have those values.

Wouldnt you rather add the `kwargs` here to make it possible to overwrite values? (same for sub_config2) 
Potentially, if you want users to be able to modify each of the three plots with different values, you'd need t think about 2 extra keywordarguments containing the config-namedtuples to be put in the respective functino